src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,96 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import java.util.Objects;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final int MIN_AGE = 18;
+    private static final int MIN_PASSWORD_LENGTH = 6;
+    private RegistrationService registrationService;
+    private User firstUser;
+    private User secondUser;
+    private User thirdUser;
+    private User fourthUser;
+    private User fifthUser;
+
+    @BeforeEach
+    void setUp() {
+        registrationService = new RegistrationServiceImpl();
+        firstUser = new User(1L, "Bob", "qwe123", 55);
+        secondUser = new User(2L, "Alice", "qwe123", 17);
+        thirdUser = new User(3L, "Bob", "567123", 18);
+        fourthUser = new User(4L, "John", "qwe56", 19);
+        fifthUser = new User(5L, "Max", "qwe569", 33);
+    }
+
+    @Test
+    void add_Ok() {
```

Comment:
what about test method naming? 
There are a lot of ways to name your test methods. The main point is that they should have informative names and be consistent along with other developers in your team. For this task use such convention: <methodUnderTest>_<state>_<expectedBehavior>; For example, if we are testing the method register with a null